pkg/authentication/externalidp/awsiam/token_caching.go

```diff
@@ -14,6 +14,8 @@ var ErrInvalidTokenFormat = fmt.Errorf("token format is invalid")
 
 const (
 	ReadWriteExecuteOwnerOnly = 0700
+	LakectlDirName            = ".lakectl"
+	CacheDirName              = "cache"
```

Comment:
1. What's the purpose of those directories? Why not just `~/.lakectl_token_cache.json`? Or just `~/.lakectl/token_cache.json` (If you really need a dir)
2. This is not the right location for such things. It belongs to lakectl code parts, not here.